_posts/2019-12-18-road-to-scala-3.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog-detail
+post-type: blog
+by: Lukas Rytz, Adriaan Moors, Martin Odersky
+title: "Scala 2 Roadmap Update: the Road to Scala 3"
+---
+
+Together with the Scala 3 team at EPFL (aka the Dotty team), led by Martin Odersky, we have decided that, rather than developing Scala 2.14, our efforts should go to Scala 3 instead.
+
+While we're very excited to shift our focus to Scala 3, we will continue to maintain Scala 2.13 to ensure the community has ample time to carefully transition to Scala 3.
```

Comment:
"While we're very excited" should probably be "While the Lightbend team is very excited", since otherwise the "we" here defaults to include Martin, who shifted his focus to Scala 3 a long time ago.